app/Http/Controllers/ProfileController.php

```diff
@@ -22,17 +23,29 @@ public function getProfileSettings()
     }
 
     /**
+     * Gets selected user's dashboard.
+     *
+     * @return Response
+     */
+    public function show($id)
+    {
+        $user     = User::with('projects')->find($id);
```

Comment:
**Code Quality - Awesome**

I love what you did with the eager loading `User::with('projects')`  :+1: